corehq/apps/reports/display.py

```diff
@@ -143,3 +144,23 @@ def append_form_name(self, name, separator):
 
 def xmlns_to_name(domain, xmlns, app_id, lang=None, separator=None, form_name=None):
     return _FormType(domain, xmlns, app_id, form_name).get_label(lang, separator)
+
+
+def xmlns_to_name_for_case_deletion(domain, form):
+    """
+    The difference between this function and the one above is that in the event that the form name
+    can't be found, this function will attempt to recreate the standard 3 part structure, rather than
+    defaulting to the xmlns url (form.xmlns). Currently only used in the case deletion workflow.
+
+    TODO: Confirm it returning the form.xmlns isn't necessary + confirm this is the format we want to display
+          for unknown form names in report tables and merge the two functions into one.
```

Comment:
Will this TODO comment be addressed prior to merging this PR?

Reply:
I'll have to ask around but as of right now, I'm think about leaving it as is, since I'm not sure it's going to be as simple as I initially thought it was going to be. If I end up not doing it, I'll create a ticket for it.